### PR TITLE
Update docs on managing translations

### PR DESCRIPTION
**Pull request by hellais**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -33,3 +33,27 @@ yarn run start
...
+From inside of `ooni/translation` to update the transifex master copy (this is
+done when edits to the master spreadsheet are done), you should run:
+```
+./update-explorer-source.sh && explorer-sync-csv-source.sh
+```
+
+Then when the translations have been done and you want to pull in the
+translated versions, run:
+```
+XXX what is the command for this?
```
Comment (hellais) [inline Readme.md:51]: @sarathms what is the command here?

Comment (sarathms) [reply Readme.md]: @hellais Done. I added a new script to the `ooni/translations` repo for this purpose in this [PR](https://github.com/ooni/translations/pull/21). We should merge it first for these instructions to be valid.

**Comment by sarathms** [comment]
I added a change that uses the directories in `ooni/translations/explorer/*` to determine the supported languages. Newly available languages will get included automatically after they are pulled from Transifex. The script `update-explorer-translations.sh` can control this by pulling translations only if they are above a minimum completion threshold.

**Review by sarathms — APPROVED** [review]
This is good to merge. But the scripts in `ooni/translations` referenced in these changes are yet to land in the repo from [this PR](https://github.com/ooni/translations/pull/21)

**Pull request merged**